### ISSUE TITLE
Onnx file not deployed

### ISSUE THREAD

**Issue by pbharat84** [dotMorten/Microsoft.Azure.Kinect.BodyTracking#2]
Sample applications are working fine as they are dependent on the SDK project as such. Using the Nuget package and trying to re-create the same sample WPF app fails with 'Couldn't create tracker'. Tried to get more info but looks like the error is happening in the Native files. 
Appreciate any help, waiting to use this one for some fun applications.

**Comment by dotMorten** [comment]
Are you specifically targeting x64? The tracking sdk only comes with 64bit libs

**Comment by pbharat84** [comment]
Yes that is correct. If I tried x86 it fails with cannot find k4abt.dll which makes sense.

**Comment by pbharat84** [comment]
So, to be clear. If I import this downloaded git project and try to build and run it, it works fine. The issue is with Nuget package.

**Comment by pbharat84** [comment]
I am very new to the c# world it may very well be me doing something wrong. I will try to give more details. I added the Unofficial.Microsoft.Azure.Kinect.BodyTracking.DotNet 1.0.0 - beta1 Nuget package.. and this piece of code `this.kinect = Device.Open();
            this.kinect.StartCameras(new DeviceConfiguration
            {
                ColorFormat = ImageFormat.ColorBGRA32,
                // ColorResolution = ColorResolution.Off,
                // DepthMode = DepthMode.NFOV_Unbinned,
                ColorResolution = ColorResolution.R1080p,
                DepthMode = DepthMode.NFOV_2x2Binned,
                SynchronizedImagesOnly = true
            });
            calibration = this.kinect.GetCalibration();
            tracker = new Tracker(calibration, Tracker.DefaultTrackerConfiguration);`
I tried to replicate the whole sample app but ended with the same error. So trying to dissect it by taking baby steps.
Please let me know if you need any additional info.

**Comment by pbharat84** [comment]
Found the issue. `dnn_model_2_0.onnx` file should be part of output directories. Not sure why this isn't part of the Nuget Package. Once you have that in place things are working fine. Thanks for your effort in putting this together.

**Comment by dotMorten** [comment]
Going to reopen since that shouldn't happen. Any chance you can share a project that reproduces the problem?
